src/test/java/com/ksm/wstbackoffice/controller/AddressControllerTest.java

```diff
@@ -0,0 +1,70 @@
+package com.ksm.wstbackoffice.controller;
+
+import com.ksm.wstbackoffice.endpoint.EndPoint;
+import io.restassured.http.ContentType;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlGroup;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.when;
+import static org.hamcrest.Matchers.equalTo;
+
+@SqlGroup({
+        @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts="classpath:sql/addressInsert.sql"),
+        @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts="classpath:sql/addressDrop.sql")
+})
+public class AddressControllerTest extends BaseController {
```

Comment:
Please rename to AddressControllerIT.
Often we put Test to the end of the unit test class and we put IT to the end of the integration test class

Reply:
done